InstallationGuide.md

```diff
@@ -1,50 +1,39 @@
 # Installation Guide
-## Product DFT
-Install from the command line:
 
-docker container run -d --name [conatainer_name] ghcr.io/catenax-ng/tx-dft-backend:[tag]
+It is necessary to inject the environment variables, credentials and URLs that can be found on application.properties file.
+For more details, please refer configuration section from [README.md](README.md)
 
+### RUN SDE backend in ArgoCD 
+ We have helm chart available for ArgoCD deployment. In deployment, if don't specified specific version, the app version/chart version is automatically picked up by ArgoCD and deployed to the environment using Helm charts.
+
+ In values.yaml you can find `default` as value for all required configuration. You need to change all those values as per your need. For reference, please refer configuration example section.
+ 
+ As part of argo CD deployment using helm chart the postgres database dependency will get provide automatic but for EDC, DigitalTwin and Portal you need to provide valid details as per configuration requirement other wise SDE service will get started with default configuration but will not work as expected.
+
+### RUN SDE Backend in k8s cluster
+#### Prerequisites
+- k8s cluster/ minikube
+- helm
+- Docker
+
+ In values.yaml you can find `default` as value for all required configuration. You need to change all those values as per your need. For reference, please refer configuration example section.
+
+ helm repo add sde-backend https://github.com/eclipse-tractusx/dft-backend/tree/main/charts
```

Comment:
Try to use markdown code with something like:

`
```sh or ```bash
`

Reply:
Sure. Requested changes made.

Reply:
Requested changes made.